qutebrowser/app.py

```diff
@@ -255,7 +256,11 @@ def process_pos_args(args, via_ipc=False, cwd=None):
             log.init.debug("Empty argument")
             win_id = mainwindow.get_window(via_ipc, force_window=True)
         else:
-            win_id = mainwindow.get_window(via_ipc)
+            if via_ipc and target_arg and target_arg != 'auto':
```

Comment:
There is a little problem:
If the target argument is not given, the running instance recieves an empty string instead of `None`, although `argparse` returns `None` from the argument parsing.
The JSON in the ipc still has `"target_arg": null`, so I think the problem comes from the PyQt signal `got_args`:
```got_args = pyqtSignal(list, str, str)```
When called with `got_args.emit(['example.org'], None, '/temp/dir')` [ipc.py:339], it looks like the slot in the `app` module recieves `(['example.org'], '', '/temp/dir')`
```
server.got_args.connect(lambda args, target_arg, cwd:
                                process_pos_args(args, cwd=cwd, via_ipc=True,
                                                 target_arg=target_arg))
```
[app.py:106]
Now that I check for `None` explicitely, it doesn't work anymore.